doc/circuitikzmanual.tex

```diff
@@ -3464,6 +3464,7 @@ \subsection{Block diagram components}
     \circuitdescbip*{highpass}{highpass}{}
     \circuitdescbip*{lowpass}{lowpass}{}
     \circuitdescbip*{allpass}{allpass}{}
+    \circuitdescbip*{bgenerator}{bgenerator}{}
```

Comment:
Please use, for example: 
`\circuitdescbip*{bgenerator}{Block generator (use \texttt{t=\dots} to specify text)}{}`